pkg/tsdb/influxdb/influxdb.go

```diff
@@ -2,37 +2,36 @@ package influxdb
 
 import (
 	"context"
+	"encoding/json"
 	"errors"
 	"fmt"
 	"net/http"
 	"net/url"
 	"path"
 	"strings"
 
+	"github.com/grafana/grafana-plugin-sdk-go/backend"
+	"github.com/grafana/grafana-plugin-sdk-go/backend/instancemgmt"
+	"github.com/grafana/grafana/pkg/components/simplejson"
 	"github.com/grafana/grafana/pkg/infra/httpclient"
 	"github.com/grafana/grafana/pkg/infra/log"
-	"github.com/grafana/grafana/pkg/models"
-	"github.com/grafana/grafana/pkg/plugins"
+	"github.com/grafana/grafana/pkg/plugins/backendplugin"
+	"github.com/grafana/grafana/pkg/plugins/backendplugin/coreplugin"
+	"github.com/grafana/grafana/pkg/registry"
 	"github.com/grafana/grafana/pkg/setting"
 	"github.com/grafana/grafana/pkg/tsdb/influxdb/flux"
+	"github.com/grafana/grafana/pkg/tsdb/influxdb/models"
+
+	"github.com/grafana/grafana-plugin-sdk-go/backend/datasource"
 )
 
-type Executor struct {
-	httpClientProvider httpclient.Provider
-	QueryParser        *InfluxdbQueryParser
-	ResponseParser     *ResponseParser
-}
+type Service struct {
+	HTTPClientProvider   httpclient.Provider   `inject:""`
+	BackendPluginManager backendplugin.Manager `inject:""`
+	QueryParser          *InfluxdbQueryParser
+	ResponseParser       *ResponseParser
```

Comment:
I think these are not properly initialized. It still works, but probably a good idea to initialize these.